stable/elasticsearch-exporter/ci/security-context.yaml

```diff
@@ -0,0 +1,6 @@
+---
+# Set default security context for kubernetes
+
+securityContext:
+  runAsNonRoot: true
```

Comment:
1/ It also requires an empty file as well to test with the default values : https://github.com/helm/charts/blob/30b5c13ae1cad13655c0e3f43c7f4cf525e76944/stable/redis/ci/default-values.yaml, so that the default values are tested

2/ In this file, you can change securityContext to be disabled.

And then let's merge this!

Reply:
@desaintmartin like this?

By the way, I am not familiar at all with those files for the CI. Do you have a link to the documentation?

Reply:
Sure! ;)
https://github.com/helm/charts/tree/master/test#providing-custom-test-values